### PR TITLE
chore: update qwel-exe

### PR DESCRIPTION
**Pull request by qwel-exe**
## Changes proposed
changed my GitHub name and few of the icons 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1024"></a>

### CODE REVIEW

**Comment by emmalearnscode** [comment]
Hi @qwel-exe I'm not sure why there are image files included in your pull request?

File: public/data/qwel-exe.json
Comment (Panquesito7) [inline public/data/qwel-exe.json:5]: I think you forgot to update this. 😅
```suggestion
  "avatar": "https://github.com/qwel-exe.png",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by qwel-exe** [comment]
> Hi @qwel-exe I'm not sure why there are image files included in your pull request?

heyyyy idk why imgbot has done that in my pr

**Comment by Panquesito7** [comment]
> > Hi @qwel-exe I'm not sure why there are image files included in your pull request?
> 
> heyyyy idk why imgbot has done that in my pr

Could you please revert the changes? Let us know if you need any help. 🙂

**Review by KendallDoesCoding — APPROVED** [review]
@emmalearnscode @Panquesito7, @ImgBotApp Just optimized the images, which I think it means reduced file size, but it didnt get added into public/data so if yall merge this it should be fine according to me.

**Comment by emmalearnscode** [comment]
> @emmalearnscode @Panquesito7, @ImgBotApp Just optimized the images, which I think it means reduced file size, but it didnt get added into public/data so if yall merge this it should be fine according to me.

It's part of the pull request which means it will affect the repo. Once the images are removed then we can get this merged.

**Comment by KendallDoesCoding** [comment]
> > @emmalearnscode @Panquesito7, @ImgBotApp Just optimized the images, which I think it means reduced file size, but it didnt get added into public/data so if yall merge this it should be fine according to me.
> 
> It's part of the pull request which means it will affect the repo. Once the images are removed then we can get this merged.

I don't understand how it will, but ok.

File: public/data/qwel-exe.json
Comment (Panquesito7) [inline public/data/qwel-exe.json:25]: Shouldn't this be lower-case?

Comment (emmalearnscode) [reply public/data/qwel-exe.json:25]: @Panquesito7 case doesn’t matter, there’s still a function for that in the app

**Comment by qwel-exe** [comment]
hey i tryed things but i am not able to remove the img we can do one thing close the pr and i can make a new one

**Comment by Panquesito7** [comment]
> hey i tryed things but i am not able to remove the img we can do one thing close the pr and i can make a new one

ImgBot should've made a PR for optimizing the images. Try reverting the PR.

**Comment by KendallDoesCoding** [comment]
> hey i tryed things but i am not able to remove the img we can do one thing close the pr and i can make a new one

Hi @qwel-exe Please download [Github Desktop](https://desktop.github.com) clone your forked branch, select branch as patch-1, go to history, see the commits imgbot made, right-click those comments, and tap revert to revert the changes. You can use github desktop on your iPad and I think phone too, so feel free to use it on any device you want :). Thanks

**Comment by KendallDoesCoding** [comment]
> > hey i tryed things but i am not able to remove the img we can do one thing close the pr and i can make a new one
> 
> Hi @qwel-exe Please download [Github Desktop](https://desktop.github.com) clone your forked branch, select branch as patch-1, go to history, see the commits imgbot made, right-click those comments, and tap revert to revert the changes. You can use github desktop on your iPad and I think phone too, so feel free to use it on any device you want :). Thanks

@qwel-exe If you need, I can make a PR in your repo, which you just need to merge to revert changes.

File: public/data/qwel-exe.json
Comment (KendallDoesCoding) [reply public/data/qwel-exe.json:25]: Still, its better to keep everything the same case, etc

Comment (KendallDoesCoding) [reply public/data/qwel-exe.json:25]: @qwel-exe , please merge this https://github.com/qwel-exe/LinkFree/pull/3

**Comment by KendallDoesCoding** [comment]
> > hey i tryed things but i am not able to remove the img we can do one thing close the pr and i can make a new one
> 
> ImgBot should've made a PR for optimizing the images. Try reverting the PR.

ImgBot, did it in a seprate branch for @qwel-exe but, @qwel-exe merged his changes in this PR's branch in commit [a826bc](https://github.com/EddieHubCommunity/LinkFree/pull/1024/commits/a826bc790e603cff68d4f4f52a69275e172758a5)

**Comment by KendallDoesCoding** [comment]
> > > hey i tryed things but i am not able to remove the img we can do one thing close the pr and i can make a new one
> > 
> > 
> > ImgBot should've made a PR for optimizing the images. Try reverting the PR.
> 
> ImgBot, did it in a seprate branch for @qwel-exe but, @qwel-exe merged his changes in this PR's branch in commit [a826bc](https://github.com/EddieHubCommunity/LinkFree/pull/1024/commits/a826bc790e603cff68d4f4f52a69275e172758a5)

@Panquesito7 ImgBot made a PR too https://github.com/qwel-exe/LinkFree/pull/1

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
@qwel-exe Please merge the following into your forked repo:

Revert changes by you merging ImgBot's PR into your forked repos PR branch: https://github.com/qwel-exe/LinkFree/pull/4

Revert changes by you merging ImgBot's PR into your forked repos main branch:
https://github.com/qwel-exe/LinkFree/pull/5

Revert ImgBot's entire commits:
https://github.com/qwel-exe/LinkFree/pull/6

Merging these 3 PR's should hopefully fix the issue.

Thanks a lot,
Kendall Does Coding

**Comment by qwel-exe** [comment]
yay thanks to @KendallDoesCoding they reverted my that pr thanks :)

**Comment by KendallDoesCoding** [comment]
> yay thanks to @KendallDoesCoding they reverted my that pr thanks :)

welcome!

**Comment by KendallDoesCoding** [comment]
@qwel-exe DO NOT MERGE THIS OR IT WILL COME AGAIN, PLEASE CLOSE IT ASAP: https://github.com/qwel-exe/LinkFree/pull/7

**Review by Panquesito7 — APPROVED** [review]

**Comment by KendallDoesCoding** [comment]
@Panquesito7 Why your not merging this? It's done right?

**Comment by emmalearnscode** [comment]
> @Panquesito7 Why your not merging this? It's done right?

We require approval by 2 maintainers, just now I’m still seeing 3 files instead on 1 as part of the PR so it’s not ready yet.

**Comment by KendallDoesCoding** [comment]
> > @Panquesito7 Why your not merging this? It's done right?
> 
> We require approval by 2 maintainers, just now I’m still seeing 3 files instead on 1 as part of the PR so it’s not ready yet.

Well, imgbot messed up everything and @qwel-exe Didn't know what it was so he merged it in, so ig dismiss this PR and make a new one?

**Comment by qwel-exe** [comment]
yess
even tho i did not merg a img bot in linkfree it still comes i will close this and make a new one or 
@KendallDoesCoding if you want a :green_square: you can make one for me

**Comment by KendallDoesCoding** [comment]
> yess even tho i did not merg a img bot in linkfree it still comes i will close this and make a new one or @KendallDoesCoding if you want a 🟩 you can make one for me

Ok, will do that in a bit, I'm creating a issue rn in this repo.

**Comment by KendallDoesCoding** [comment]
@qwel-exe Made #1073 you can close this, I have a suggestion though, will review code in #1073 and let me know if that works.

**Comment by KendallDoesCoding** [comment]
> @qwel-exe Made #1073 you can close this, I have a suggestion though, will review code in #1073 and let me know if that works.

Actually, don't close this yet, merge https://github.com/qwel-exe/LinkFree/pull/4 and see if it fixes the issue, if it doesn't then close.

**Pull request closed without merging**
